roles/contiv_network/tasks/main.yml

```diff
@@ -20,21 +20,35 @@
 - name: download netmaster and netplugin
   get_url:
     validate_certs: "{{ validate_certs }}"
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-12-11-2015.20-54-40.UTC/netplugin-v0.0.0-12-11-2015.20-54-40.UTC.tar.bz2
+    url: https://github.com/contiv/netplugin/releases/download/v0.1-01-28-2016.03-55-05.UTC/netplugin-v0.1-01-28-2016.03-55-05.UTC.tar.bz2
```

Comment:
@DivyaVavili - the option is changed to 
```
  -plugin-mode string
        plugin mode docker|kubernetes (default "docker")
```
however what used to be `dock-plugin` is now `-plugin-mode docker` which is the default when `-plugin-mode` is not specified.